user/message.c

```diff
@@ -3184,7 +3184,8 @@ static BOOL CALLBACK enum_scrollbar_proc(HWND hwnd, LPARAM lp)
 static LRESULT defwindow_proc_callback(HWND hwnd, UINT msg, WPARAM wp, LPARAM lp,
     LRESULT *result, void *arg)
 {
-    _LeaveWin16Lock();
+    DWORD count;
```

Comment:
It was a workaround for installers that do not work properly if NULL is returned in FindWindow immediately after WinExec.
It seems to work without this workaround, since there is an idle_event.

Reply:
Thanks, good to know.